packages/create-fuels/src/cli.ts

```diff
@@ -113,6 +115,16 @@ function writeEnvFile(envFilePath: string, programsToInclude: ProgramsToInclude)
   writeFileSync(envFilePath, newFileContents);
 }
 
+async function promptForFuelUpInstall() {
+  const shouldInstallFuelUp = await prompts({
+    type: 'confirm',
+    name: 'shouldInstallFuelUp',
+    message: 'fuelup is not installed. Would you like us to install it for you?',
```

Comment:
Should we surround `fuelup` with backticks?

```suggestion
    message: "You don't have `fuelup` installed. Would you like us to install it for you?",
```